install-deps.sh

```diff
@@ -167,6 +167,8 @@ function activate_virtualenv() {
     . $env_dir/bin/activate
 }
 
+test "$1" = "skip-virtualenv" && exit 0
```

Comment:
If it is made the default all environment that rely on install-deps.sh to run tests will have to be modified. If we don't make it the default all environments that fail to run because install-deps.sh will have to add this option. Unless I'm mistaken there only are two environments that motivate this change : the gitbuilders and the jenkins builders and both can be modified to add this argument. By contrast there is an unknown number of environment (most of which are out of our control) that depend on the current behavior.